_core/regist/inc/tripcode.php

```diff
@@ -18,9 +18,12 @@ function format($input) {
         list($name, $trip, $secure) = explode("#",$input,3);
 
         if ($secure || $trip)
-            $trip = ($secure) ? Tripcode::secure($secure) : Tripcode::normal($trip);
+            $trip = ($secure) ? $this->secure($secure) : $this->normal($trip);
```

Comment:
While nothing wrong with this, it was for using them statically.

Instead of making a new instance of Tripcode previously we could just load the *Tripcode* class then `Tripcode::format(blah)`. However `$this` requires it made an instance.